db/seeds.rb

```diff
@@ -64,3 +68,27 @@
   { name: 'Chitenges' }, { name: 'Khangas' }, { name: 'Lesos' },
   { name: 'Mantas' }, { name: 'Rebozos' }
 ])
+
+# Importing carriers
+path = File.join Rails.root, 'public', 'inventory.csv'
```

Comment:
Putting this file in the `public` directory makes it available for anyone to download it from our site if they know it exists. What if we tucked it somewhere else so it wasn't publicly accessible? Maybe `db/data/inventory.csv`?

Reply:
thanks, I moved the file to db/data